RELEASE/scripts/autoscend/paths/avant_guard.ash

```diff
@@ -0,0 +1,173 @@
+boolean in_ag()
+{
+	return my_path() == $path[Avant Guard];
+}
+
+void ag_initializeSettings()
+{
+	if(in_ag())
+	{
+		set_property("auto_100familiar", "Burly Bodyguard");
+		set_property("auto_skipNuns", true);
+	}
+}
+
+void ag_pulls()
+{
+	if(in_ag())
+	{
+		//SUPER helpful for gremlins
+		if(storage_amount($item[mini kiwi invisible dirigible]) > 0 && auto_is_valid($item[mini kiwi invisible dirigible]))
```

Comment:
this is tradable so could buy one if it's not in storage, it's not expensive. can be done later if you want to merge this

Reply:
pullXWhenHaveY will try to buy if the price is low enough so this is actually already taken care of

Reply:
would need to remove the storage_amount check to allow pullXWhenHaveY to buy it if we don't have it already